### PR TITLE
docs: OpenAPI Client tag

### PR DESCRIPTION
**Pull request by chriswk**
### What
Adds documentation for open api endpoints tagged with `Client`.

### CODE REVIEW

File: src/lib/openapi/spec/client-feature-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/client-feature-schema.ts:31]: While this is funny, I'm afraid it could also be misleading. I'd suggest something (sadly) more mundane. How about this? It links together with the feature name example

```suggestion
            example: 'Enable the new payment flow',
```

File: src/lib/openapi/spec/client-feature-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/client-feature-schema.ts:36]: Should this be 'environment' or 'API key'? 

```suggestion
                'Whether the feature flag is enabled for the current environment or not. This is ANDed with the evaluation results of the strategies list, so if this is false, the evaluation result will always be false',
```

File: src/lib/openapi/spec/client-feature-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/client-feature-schema.ts:41]: Doesn't this have to be explicitly set by the users? I thought Unleash only said features *may* be stale, but that it required a user to mark them as such?

```suggestion
                'Whether this feature toggle has become [stale](https://docs.getunleash.io/reference/technical-debt#stale-and-potentially-stale-toggles "Stale toggles") or not.',
```

File: src/lib/openapi/spec/client-features-query-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/client-features-query-schema.ts:18]: Interesting. Why is it a list of lists here? It's probably how the API is designed, but it seems weird.

File: src/lib/openapi/spec/client-features-query-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/client-features-query-schema.ts:11]: Hmm maybe something like this to make it clearer that it excludes features without the tags?

```suggestion
            description: 'Only return features tagged with at least one of the provided tags',
```

File: src/lib/openapi/spec/client-features-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/client-features-schema.ts:22]: Discussion: should this property be an enum instead of a number? Maybe, maybe not. Do you have any thoughts? If all we have is 1 and 2, then it's not very helpful to put 564, for instance. Not very important in a response schema, but interesting none the less.

File: src/lib/openapi/spec/override-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/override-schema.ts:18]:
```suggestion
            description: 'The list of values that should trigger this override. If the context field (in `contextName`) has one of these values, activate the override.',
```

File: src/lib/openapi/spec/variant-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/variant-schema.ts:9]: I feel like saying "our **excellent** documentation" is a little too ... grand (being a Norwegian and all).
But also, the link should explain what bit of the docs its pointing to. And we should also try to avoid using the term "see" to be more inclusive towards vision-impaired people 💁🏼 How about this?

```suggestion
        'A variant allows for further separation of users into segments. Check out the [variants documentation](https://docs.getunleash.io/reference/feature-toggle-variants#what-are-variants) for a more detailed description',
```

File: src/lib/openapi/spec/variant-schema.ts
Comment (thomasheartman) [inline src/lib/openapi/spec/variant-schema.ts:14]:
```suggestion
            description: 'The variant's name. The name will be unique for this feature toggle',
```

File: src/lib/routes/client-api/register.ts
Comment (thomasheartman) [inline src/lib/routes/client-api/register.ts:46]:
```suggestion
                        'Register a client SDK with Unleash. SDKs call this endpoint on startup to tell Unleash about their existence. Used to track custom strategies in use as well as SDK versions.',
```

**Review by thomasheartman — APPROVED** [review]
Very nice work here! A lot of changes, so it's super good to see it done 💯  I've added a few suggestions that should probably be looked at before merging, but I trust you to do that without needing a second approval. Well done 😄

File: src/lib/openapi/spec/client-application-schema.ts
Comment (chriswk) [reply src/lib/openapi/spec/client-application-schema.ts:37]: Agreed. Thanks for the suggestion, applied

**Pull request merged**
